apps/site/assets/js/test/event-page-setup__test.js

```diff
@@ -39,10 +39,38 @@ const eventsHubHTML = `
     </div>
     `;
 
+function triggerScrollEvent(clock) {
+  document.dispatchEvent(new window.Event("scroll"));
+  clock.tick(); // fast forward set timeout (gets past window.requestAnimationFrame)
+}
+
 describe("setupEventsPage", () => {
```

Comment:
Looks like it's a little bit of a pain to trigger these events so glad you found a way to test them! Thanks for adding all these comments for better understanding 🙏